ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java

```diff
@@ -3175,4 +3175,7 @@
     @LogMessage(level = WARN)
     @Message(id = 508, value = "Failed to persist timer's state %s due to %s")
     void exceptionPersistTimerState(Timer timer, Exception e);
+
+    @Message(id = 509, value = "Client mapping is bound to '%s' in cluster mode, please specify client mappings in socket-binding: %s")
```

Comment:
What do you mean by "in cluster mode"? This is problem in any mode.

Reply:
It means the server is starting with HA profile. The cluster information with client mappings will be sent to ejb client side in this case, then ejb client will try to use the bound IP(`0.0.0.0`) to locate the ejb target. If the server starts without HA profile, the server [won't send the problematic client mappings to ejb client](https://github.com/wildfly/wildfly/blob/master/ejb3/src/main/java/org/jboss/as/ejb3/remote/AssociationImpl.java#L451), and ejb client will only discover the ejb targets using the configured remoting connection address. so there is no problem in this case.

Reply:
I see what you mean now, when there is only a single server with which the client is configured, the client is never sent the topology, so its only problem when the topology is sent to the client which only happens with HA profile. Though, non-clustered EJBs in production is not a common use case, so I don't think we need to make that explicit.

Reply:
> Though, non-clustered EJBs in production is not a common use case, so I don't think we need to make that explicit.

I think the Jira is in `minor` priority because the issue is not a common use case and bound to `0.0.0.0` is not recommended in production. :)